ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -54,9 +56,22 @@ message CreatedEvent {
   Value contract_key = 7;
 
   // The arguments that have been used to create the contract.
-  // Required
+  // Set if there was a match by template_id in the given transaction filter.
+  // Optional
   Record create_arguments = 4;
```

Comment:
Do I understand it correctly, that this will be never populated for an interface subscription? Can it be a use-case where the original argument is useful despite of an interface subscription?

Reply:
I read it as this _or_ `include_create_arguments_blob` was true. But maybe the latter should simply be removed.

Reply:
Which I now see doesn't make sense. Never mind me, carry on

Reply:
Yes, this should never be populated for an interface subscription. I don’t see why you would subscribe to an interface and get the template payload.